custom_mappings.yaml

```diff
@@ -191,6 +365,8 @@ entries:
         anilist-id: 104308
       - season: 2
         anilist-id: 124136
+      - season: 3
+        anilist-id: 154473
```

Comment:
This season 3 is under the wrong title. Move up one under Arifureta

Reply:
Fixed